src/Dapr.Workflow/DaprWorkflowClient.cs

```diff
@@ -0,0 +1,310 @@
+        /// does <b>not</b> resume the workflow if suspend was successful.
+        /// </param>
+        /// <returns>A task that completes when the suspend has been committed to the backend.</returns>
+        public Task SuspendWorkflowAsync(
+            string instanceId,
+            string? reason = null,
+            CancellationToken cancellation = default)
+        {
+            return this.innerClient.SuspendInstanceAsync(instanceId, reason, cancellation);
+        }
+
+        /// <summary>
+        /// Resumes a workflow instance that was suspended via <see cref="SuspendWorkflowAsync(string, string, CancellationToken)" />.
+        /// </summary>
+        /// <param name="instanceId">The instance ID of the workflow to resume.</param>
+        /// <param name="reason">The optional resume reason.</param>
+        /// <param name="cancellation">
+        /// A <see cref="CancellationToken"/> that can be used to cancel the resume operation. Note, cancelling this token
+        /// does <b>not</b> re-suspend the workflow if resume was successful.
+        /// </param>
+        /// <returns>A task that completes when the resume has been committed to the backend.</returns>
+        public Task ResumeWorkflowAsync(
+            string instanceId,
+            string? reason = null,
+            CancellationToken cancellation = default)
+        {
+            return this.innerClient.ResumeInstanceAsync(instanceId, reason, cancellation);
+        }
+
+        /// <summary>
+        /// Purges workflow instance state from the workflow state store.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// This method can be used to permanently delete workflow metadata from the underlying state store,
+        /// including any stored inputs, outputs, and workflow history records. This is often useful for implementing
+        /// data retention policies and for keeping storage costs minimal. Only workflow instances in the
+        /// <see cref="WorkflowRuntimeStatus.Completed"/>, <see cref="WorkflowRuntimeStatus.Failed"/>, or
+        /// <see cref="WorkflowRuntimeStatus.Terminated"/> state can be purged.
+        /// </para>
+        /// </remarks>
+        /// <param name="instanceId">The unique ID of the workflow instance to purge.</param>
+        /// <param name="cancellation">
+        /// A <see cref="CancellationToken"/> that can be used to cancel the purge operation.
+        /// </param>
+        /// <returns>
+        /// Returns a task that completes when the purge operation has completed. The value of this task will be
+        /// <c>true</c> if the workflow state was found and purged successfully; <c>false</c> otherwise.
+        /// </returns>
+        public async Task<bool> PurgeInstanceAsync(string instanceId, CancellationToken cancellation = default)
+        {
+            PurgeResult result = await this.innerClient.PurgeInstanceAsync(instanceId, cancellation);

```

Comment:
Note that purge isn't yet implemented by the engine and won't work until that implementation is complete.

Reply:
I think it's better to not add a comment here since the purge functionality will be available before we ship this SDK version (Ryan and I are actively working on it now). Otherwise, we'd need another PR in a week or two to remove the doc/comment.